packages/phone-number-privacy/common/src/interfaces/domains.ts

```diff
@@ -0,0 +1,26 @@
+export interface SequentialDelayDomain {
+  name: 'Sequential Delay Domain'
+  version: number
+  stages: SequentialDelayStage[]
+  // Optional public key of a against which signed requests must be authenticated.
+  // In the case of Cloud Backup, this will be a one-time key stored with the ciphertext.
+  publicKey?: string
+  // Optional string to distinguish the output of this domain instance from
+  // other SequentialDelayDomain instances
+  salt?: string
+}
+
+export interface SequentialDelayStage {
+  // How many seconds each batch of attempts in this stage is delayed with
+  // respect to the timer.
+  delay: number
+  // Whether the timer should be reset between attempts during this stage.
```

Comment:
Since it's between, but also on the last attempt in the stage...
```suggestion
  // Whether the timer should be reset after each attempt during this stage.
```